controllers/orderController.js

```diff
@@ -40,6 +42,7 @@ const createOrder = async (req, res) => {
       product: _id,
     };
     //add item to order
+    // AKOSREVIEW, in this case, could also just do push since we're mutating the original array anyway
     orderItems = [...orderItems, singleOrderItem]; //whatever items we have.. with each iteration add new  singleOrderItem
```

Comment:
I did some further research, and I think using the spread operator as you did is actually more performant. But my initial thinking is that you could instead do `orderItems.push(singleOrderItem)` to just add the single order item to the order items list each time.

Reply:
@akosasante thank you!! I will leave option with push it works perfectly